test/types.cpp

```diff
@@ -50,8 +50,11 @@
 #include <nil/marshalling/compile_control.hpp>
 #include <nil/marshalling/units.hpp>
 #include <nil/marshalling/version.hpp>
+#include <nil/marshalling/inference.hpp>
 
 #include <nil/marshalling/algorithms/pack.hpp>
+#include <nil/marshalling/algorithms/disperse.hpp>
+#include <nil/marshalling/algorithms/marshal.hpp>
```

Comment:
Still kinda think this name (`marshal`) is questionable.

Reply:
Fixed.